### DISCUSSION TITLE
Game crashed after trying to spectate a multiplayer room

### DISCUSSION THREAD

**Discussion by Gonzalo-Bruna** [ppy/osu#19055] [Q&A] [answered]
after I pressed the spectate button it crashed right away

[legacy-ipc.log](https://github.com/ppy/osu/files/9075716/legacy-ipc.log)
[network.log](https://github.com/ppy/osu/files/9075717/network.log)
[performance.log](https://github.com/ppy/osu/files/9075718/performance.log)
[performance-audio.log](https://github.com/ppy/osu/files/9075719/performance-audio.log)
[performance-draw.log](https://github.com/ppy/osu/files/9075720/performance-draw.log)
[performance-input.log](https://github.com/ppy/osu/files/9075721/performance-input.log)
[performance-update.log](https://github.com/ppy/osu/files/9075722/performance-update.log)
[runtime.log](https://github.com/ppy/osu/files/9075723/runtime.log)
[updater.log](https://github.com/ppy/osu/files/9075724/updater.log)
[database.log](https://github.com/ppy/osu/files/9075725/database.log)

**Comment by peppy** [comment, marked as answer]
https://github.com/ppy/osu/issues/19053
